### PR TITLE
Auto Select Fight/Player if in WCL URL

### PR DESCRIPTION
**Pull request by awlego**
Implemented auto selecting of fight and player if provided in the passed URL.

### CODE REVIEW

File: src/Main/App.js [hunk 1]
```diff
@@ -78,7 +78,11 @@ class App extends Component {
   }
 
   getPlayerFromReport(report, playerName) {
-    return report.friendlies.find(friendly => friendly.name === playerName);
+    const fetchByNameAttempt = report.friendlies.find(friendly => friendly.name === playerName);
+    if (!fetchByNameAttempt) {
+      return report.friendlies.find(friendly => friendly.id === parseInt(playerName, 10));
```
Comment (MartijnHols) [inline src/Main/App.js]: `parseInt()` is outdated JS, please use `Number()` instead.

**Comment by MartijnHols** [comment]
Fixes #444

File: src/Main/App.js (hunk 1 above)
Comment (awlego) [reply src/Main/App.js]: I originally was going to include fight name and player name, but it requires blocking on the report be loaded which is certainly possible, but felt less clean. I can certainly try that route again.

I'll update to use Number().

**Comment by awlego** [comment]
Updated to use Number().

**Comment by MartijnHols** [comment]
Thanks! I would prefer an approach that shows the fight and player names in the URL, but it's not required.

**Comment by awlego** [comment]
Thanks for the approval. I have some free time this weekend so I'll see if I can get the actual names in another PR.

**Pull request merged**